src/scss/components/content/_index.scss

```diff
@@ -6,9 +6,6 @@
 	@if output(content-default-link) {
 		a:not([class])[href] {
 			color: currentColor;
-			text-decoration: none;
```

Comment:
moet die niet underline worden?

Reply:
kan wel hoor, maar is toch default browser gedrag?